frontend/src/components/events/atoms/flex/flex.css

```diff
@@ -0,0 +1,32 @@
+.flex {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 20px;
+}
+
+.flex-item {
+  display: flex;
+  flex: 0 1 27%;
+  justify-content: center;
+  align-items: center;
```

Comment:
we may want to left align upcoming events since there may not be many but not sure